utils/dataset_manifest/core.py

```diff
@@ -611,6 +665,13 @@ def create(self, content=None, _tqdm=None):
 
         self.set_index()
 
+    def get_first_not_empty_item(self) -> ImageProperties:
+        for _, image in self:
+            if image:
+                return image
+
+        raise InvalidManifestError('Manifest is empty')
```

Comment:
It feels like this API change should be at least reflected in the PR description.

Reply:
why? I guess it is internal API and it's not worth mentioning that it was added

Reply:
Ok, I see it's handled internally, so it can be ignored.